nibabel/tests/test_proxy_api.py

```diff
@@ -131,6 +131,22 @@ def validate_asarray(self, pmaker, params):
         # Shape matches expected shape
         assert_equal(out.shape, params['shape'])
 
+    def validate_get_scaled(self, pmaker, params):
+        # Check proxy returns expected array from asarray
+        prox, fio, hdr = pmaker()
+        out = prox.get_scaled()
+        assert_array_equal(out, params['arr_out'])
+        assert_dt_equal(out.dtype, params['dtype_out'])
+        # Shape matches expected shape
+        assert_equal(out.shape, params['shape'])
+
+        for dtype in np.sctypes['float']:
```

Comment:
This could easily be extended with non-float types, and the checks would still be valid. Not sure if that's a case we want to specifically support...

Reply:
Any reason not to check this? Just to avoid the possibility of a silent fail.